src/components/WorkOrder/WorkOrderDetails.test.js

```diff
@@ -102,6 +102,30 @@ describe('WorkOrderDetails component', () => {
       )
       expect(asFragment()).toMatchSnapshot()
     })
+    it('should not render Multibutton if work order is closed', () => {
+      const { asFragment } = render(
+        <UserContext.Provider value={{ user: agent }}>
+          <WorkOrderDetails
+            propertyReference={props.property.propertyReference}
+            workOrder={
+              new WorkOrder({
+                ...workOrderData,
+                status: 'No Access',
```

Comment:
The four new unit tests in this file only seem to differ by their `status` - is that correct? If so I can't see how that would change the behaviour of the component under test. It seems to me that we can safely remove all but one of the tests, remove the status, and just set `closedDated`.